chained/tlsmasq_impl.go

```diff
@@ -142,6 +148,7 @@ func (impl *tlsMasqImpl) dialServer(op *ops.Op, ctx context.Context) (net.Conn,
 	if impl.tlsClientHelloSplitting {
 		tcpConn = hellosplitter.Wrap(tcpConn, splitClientHello)
 	}
+	tcpConn = tlsFragConn(tcpConn, impl.proxyConfig)
```

Comment:
Same comment as on the HTTPS transport:
https://github.com/getlantern/flashlight/pull/1347#discussion_r1456459403